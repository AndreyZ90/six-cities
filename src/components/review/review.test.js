import React from 'react';
import TestRenderer from 'react-test-renderer';

import Review from '@/components/review/review';

const count = 10;
const childrenNode = <div className="some-children-node">Review children node</div>;

describe(`Review component snapshot`, () => {
  test(`Should correctly render Review component`, () => {
    const tree = TestRenderer.create(
        <Review count={count}>
          {childrenNode}
        </Review>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
