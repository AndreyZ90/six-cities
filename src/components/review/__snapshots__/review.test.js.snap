// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Review component snapshot Should correctly render Review component 1`] = `
<section
  className="property__reviews reviews"
>
  <h2
    className="reviews__title"
  >
    Reviews Â· 
    <span
      className="reviews__amount"
    >
      10
    </span>
  </h2>
  <div
    className="some-children-node"
  >
    Review children node
  </div>
</section>
`;
